defaultPnorLayoutSingleSide.xml

```diff
@@ -174,6 +174,8 @@ Layout Description
         <physicalOffset>0x961000</physicalOffset>
         <physicalRegionSize>0x100000</physicalRegionSize>
         <side>A</side>
+        <xz/>
+        <xzSize>0x100000</xzSize>
```

Comment:
I thought about this a little more.  The BMC doesn't currently update the TOC correctly, so I guess it is reasonable for us to hard-code this rather than make it dynamic.  But, it seems like we should set it to somewhere between 2x and 8x the partition size.  I guess skiboot isn't allowed to be bigger than 1MB uncompressed anyhow though (so we can support existing machines) so maybe it doesn't matter much.